Currently Loaded Modulefiles:
  1) modules/3.2.11.4
  2) altd/2.0
  3) darshan/3.2.1
  4) craype-network-aries
  5) udreg/2.3.2-7.0.1.1_3.61__g8175d3d.ari
  6) ugni/6.0.14.0-7.0.1.1_7.63__ge78e5b0.ari
  7) dmapp/7.1.1-7.0.1.1_4.72__g38cf134.ari
  8) gni-headers/5.0.12.0-7.0.1.1_6.46__g3b1768f.ari
  9) xpmem/2.2.20-7.0.1.1_4.28__g0475745.ari
 10) job/2.2.4-7.0.1.1_3.55__g36b56f4.ari
 11) dvs/2.12_2.2.167-7.0.1.1_17.11__ge473d3a2
 12) alps/6.6.58-7.0.1.1_6.30__g437d88db.ari
 13) rca/2.2.20-7.0.1.1_4.74__g8e3fb5b.ari
 14) craype-haswell
 15) cray-mpich/7.7.10
 16) Base-opts/2.4.139-7.0.1.1_4.85__gbb799dd.ari
 17) gcc/8.3.0
 18) craype/2.6.2
 19) cray-libsci/19.06.1
 20) pmi/5.0.14
 21) atp/2.1.3
 22) PrgEnv-gnu/6.0.5
Warming up caches and configs on computing nodes ....

Computing your density cubes for snap 99

Traceback (most recent call last):
  File "create_density_cubes.py", line 58, in <module>
    compute_snap(snap_num, sys.argv[1])
  File "create_density_cubes.py", line 47, in compute_snap
    full_field = mesh.compute()
  File "/dev/shm/local/lib/python3.7/site-packages/nbodykit/base/mesh.py", line 250, in compute
    return self._paint_XXX(mode=mode, Nmesh=Nmesh)
  File "/dev/shm/local/lib/python3.7/site-packages/nbodykit/base/mesh.py", line 289, in _paint_XXX
    var = self.to_field(mode=actions[0][0])
  File "/dev/shm/local/lib/python3.7/site-packages/nbodykit/base/mesh.py", line 235, in to_field
    real = self.to_real_field()
  File "/dev/shm/local/lib/python3.7/site-packages/nbodykit/source/mesh/species.py", line 157, in to_real_field
    species_mesh.to_real_field(out=real, normalize=False)
  File "/dev/shm/local/lib/python3.7/site-packages/nbodykit/source/mesh/catalog.py", line 341, in to_real_field
    c1 = real1.r2c()
  File "/dev/shm/local/lib/python3.7/site-packages/pmesh/pm.py", line 661, in r2c
    out = TransposedComplexField(self.pm)
  File "/dev/shm/local/lib/python3.7/site-packages/pmesh/pm.py", line 1086, in __init__
    Field.__init__(self, pm, base)
  File "/dev/shm/local/lib/python3.7/site-packages/pmesh/pm.py", line 240, in __init__
    self.value = base.view_output()
  File "pfft/core.pyx", line 792, in pfft.core.LocalBuffer.view_output
numpy.core._exceptions.MemoryError: Unable to allocate 32.0 GiB for an array with shape (2048, 2048, 1025) and data type complex64
srun: error: nid12974: task 0: Exited with exit code 1
srun: launch/slurm: _step_signal: Terminating StepId=52253264.2
srun: Job 52253264 step creation temporarily disabled, retrying (Requested nodes are busy)
srun: Step created for job 52253264
